konflux-ci/ui/core/proxy/proxy.yaml

```diff
@@ -228,22 +228,22 @@ kind: Service
 metadata:
   labels:
     app: proxy
+  annotations:
+    service.beta.openshift.io/serving-cert-secret-name: serving-cert
```

Comment:
Does it make sense to have it in this repo?
We don't have anything to test it here.
Might make better sense to patch it when deployed on openshift.

Reply:
removed